index.html

```diff
@@ -13,18 +13,16 @@
 <body>
     <header>
         <nav>
-            <div><a href="index.html">Leo</a>
-            </div>
-            <div>
-                <div>
-                    <ul class="nav-items">
-                        <li><a class="nav-link" href="#">Portfolio</a></li>
-                        <li><a class="nav-link" href="#about">About</a></li>
-                        <li><a class="nav-link" href="#contact">Contact</a></li>
-                        <li><a class="nav-link" href="mailto: contact.popey17@gmail.com"><i class="fa-regular fa-envelope"></i></a></li>
-                    </ul>
-                </div>
-                <div><a href="#"><i class="fa-solid fa-bars"></i></a></div>
+            <a href="index.html">Leo</a>
+            <div><i class="fa-solid fa-bars menu-btn"></i></div>
+            <div class="nav-content">
+                <ul class="nav-items">
+                    <li><a class="nav-link" href="#">Portfolio</a></li>
+                    <li><a class="nav-link" href="#about">About</a></li>
+                    <li><a class="nav-link" href="#contact">Contact</a></li>
+                    <li><a class="nav-link" href="mailto: contact.popey17@gmail.com"><i
+                                class="fa-regular fa-envelope"></i></a></li>
+                </ul>
```

Comment:
### Kindly follow the design guidelines in Figma for the menu:
- [ ] The menu should take the entire width of the page.
- [ ] Please follow the same `font-family`, `font-size`, and `spacing` for the menu items.
- [ ] Please add the horizontal line between the menu items.
- [ ] Please add the `X` button which is responsible for closing the menu.
- [ ] Please remove the envelope icon.

| Your design | Figma design |
| --- | --- |
|![image](https://user-images.githubusercontent.com/24703171/223086105-d6a81c1a-119f-4983-bca7-835f7c537066.png) | ![image](https://user-images.githubusercontent.com/24703171/223085997-703c6f14-0eff-4283-b5be-138c94cc8c09.png) |